viite-UI/src/view/MapView.js

```diff
@@ -64,8 +64,9 @@
     }, this);
 
     var setCursor = function(tool) {
-      var cursor = {'Select': 'default', 'Add': 'crosshair', 'Cut': 'pointer', 'Copy': 'copy'};
-      $('.olMap').css('cursor', cursor[tool]);
+      var cursor = {'Select': 'default', 'Add': 'crosshair', 'Cut': 'crosshair', 'Copy': 'copy'};
+      // $('.olMap').css('cursor', cursor[tool]);
```

Comment:
Please assess that this commented code is needed.

Reply:
yep needs cleanup